homeassistant/components/stream/worker.py

```diff
@@ -127,7 +131,7 @@ def __init__(
         self,
         hass: HomeAssistant,
         video_stream: av.video.VideoStream,
-        audio_stream: av.audio.stream.AudioStream | None,
+        audio_stream: av.audio.AudioStream | None,
```

Comment:
`AudioStream` is available from `av.audio` as well.
https://github.com/uvjustin/PyAV/blob/v10.1.2/av/audio/__init__.py